install_config/router/default_haproxy_router.adoc

```diff
@@ -207,6 +207,28 @@ $ oadm router region-west -o yaml --images=myrepo/somerouter:mytag \
 ====
 endif::[]
 
+[[concurrent-connections]]
+Setting the Maximum Number of Concurrent Connections::
+
+The default maximum number of connections that can be handled by the router is 2000. 
+You can change that limit depending on your needs.  Too few connections will prevent the
+health check from working which causes unneeded restarts. The system needs to be configured
+to support the number of connections.  The limits shown in 'sysctl fs.nr_open' and 'sysctl fs.file-max'
```

Comment:
Well, not exactly. max-connections is the size of the pool. healthz and users all take from the pool, so there need to be enough satisfy all the users of the pool. This is fuzzy and must "big enough". Sigh.